src/cljs/main/broadfcui/page/workspace/method_configs/method_config_editor.cljs

```diff
@@ -80,7 +80,7 @@
                                        :style {:width 500}}))
            (when-not editing?
              (or field-value [:span {:style {:fontStyle "italic"}} "No value entered"]))]
-          (when error
+          (when (and error (not optional))
             [:div {:style {:padding "0.5em" :marginBottom "0.5rem"
```

Comment:
This fixes the following bug:
Optional inputs that are empty had error messages underneath them